src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkBooking.kt

```diff
@@ -23,6 +24,9 @@ class VideoLinkBooking(
   var courtName: String? = null,
   var courtId: String? = null,
   val madeByTheCourt: Boolean? = true,
+  val agencyId: String,
```

Comment:
maybe call this prisonId? (just to avoid confusion with courtId which is also an agencyId)